lint/witnesslint/__init__.py

```diff
@@ -0,0 +1,7 @@
+# This file is part of sv-witnesses repository: https://github.com/sosy-lab/sv-witnesses
+#
+# SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+__version__ = "DEV"
```

Comment:
We didn't have an explicit version number until now, so we might as well call this version 1.0 .

Since there will likely be some changes to the linter in the near future we could also call the current version on `main` 1.0 and update this as changes are made (e.g. branch `linter-improvements` introduces checks specific to SV-COMP, which would justify a version bump IMO). We would then release the most recent version as a package and use the corresponding version number.

I am actually in favor of the second option, because having a package available for the linter is not a pressing matter and there is not really a point in releasing a version now when there will be forseeable changes soon.

Reply:
Since our current version number is `1.2-dev` after #57, this PR bumps the version to `1.3` such that we can then release a clean version.

Somewhat problematic is the fact that poetry does not seem to offer an equivalent to setuptools' `attr: ` syntax for extracting the version number from the `__init__.py`, meaning that we now have to manually update the version number in two places.